test/index-test.js

```diff
@@ -1,20 +1,61 @@
+// @flow
 import { describe, it } from 'mocha'
-import { eq } from '@briancavalier/assert'
-import { at, chain, mergeArray, runEffects, take, tap } from '@most/core'
+import { eq, assert } from '@briancavalier/assert'
+import { at, mergeArray, runEffects, take, tap } from '@most/core'
+import { newDefaultScheduler, delay } from '@most/scheduler'
 import { hold } from '../src/index'
-import { newDefaultScheduler } from '@most/scheduler'
 
-export const drain = s => runEffects(s, newDefaultScheduler())
-
-export const collect = s => {
+const collect = (stream, scheduler) => {
   const into = []
```

Comment:
I think `into` is a poor name. How about `eventValues`?

Reply:
:+1: I like it

Reply:
Done in 9c58d71